src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -1,4 +1,16 @@
 ﻿@page "/component"
+
+<style>
+    .bit-nav-wrapped-link a {
+        white-space: normal;
+        line-height: inherit;
+    }
+    .bit-nav-simple-link a {
+        white-space: nowrap;
+        line-height: 44px;
+    }
+</style>
```

Comment:
please do not define styles in this file.
move all styles to the Component.razor.scss file.